pkg/storage/testdata/mvcc_histories/delete_range_predicate

```diff
@@ -47,7 +47,7 @@ meta: "i"/0,0 -> txn={id=00000001 key=/Min iso=Serializable pri=0.00000000 epo=0
 data: "i"/7.000000000,0 -> /BYTES/i7
 
 # Writing next to or above point keys and tombstones should work.
-run stats ok
```

Comment:
curious: with your log-ops flag, do any of these unit tests fail without your change?

Reply:
Yup!  The reason I needed to add the extra test case was specifically to get a test that would produce incorrect range key operations.  Some of the existing tests already produce incorrect point deletion operations.